README.md

```diff
@@ -130,52 +130,46 @@ git clone https://github.com/pytti-tools/pytti-notebook
 
       ```
       # Download this codebase
-      git clone https://github.com/pytti-tools/pytti-core
-
-      # Download research code from reference repositories
-      git clone https://github.com/openai/CLIP
-      git clone https://github.com/CompVis/taming-transformers
-
-      # Download research code modified to integrate better with PYTTI
-      git clone https://github.com/pytti-tools/AdaBins
-      git clone https://github.com/pytti-tools/GMA
+      git clone --recurse-submodules https://github.com/pytti-tools/pytti-core
```

Comment:
if we add pytti-core as a submodule under pytti-notebook, we can recurse there and eliminate another setup step.

Reply:
It's not a bad idea, but it only eliminates that step for notebook users, which would necessitate at least two sets of setup instructions. If we can get the local notebook to auto-setup too, then game on. Otherwise... probably wanna keep it with the one set of setup instructions